1.7/administration/security-and-authentication/managing-authentication.md

```diff
@@ -0,0 +1,129 @@
+---
+post_title: Managing Authentication
+menu_order: 1
+---
+Authentication is managed in the DC/OS web interface.
+
+You can authorize individual users. You can grant access to users who are local or remote to your datacenter. 
+
+The DC/OS user database is persisted in ZooKeeper running on the master nodes in znodes under the path `/dcos/users`.
+
+Tokens sent to DC/OS in a HTTP Authorization header must be of the format `token=<token>` instead of the `Bearer <token>`. Both formats
+will be supported in a future release.
+
+# User management
```

Comment:
@pyronicide I'd say they should merged. This has some details the others doesn't. And vice versa. What do you think?